backend/src/kot/models.py

```diff
@@ -1,14 +1,17 @@
+from django.core.validators import EmailValidator, validate_email, MinLengthValidator
 from django.db import models
 
+
 # Create your models here.
 
 class User(models.Model):
-    user_id = models.AutoField(primary_key=True)
-    username = models.CharField(max_length=30)
-    password = models.CharField(max_length=30)
+    email = models.CharField(primary_key=True, max_length=50, default="sample@drexel.edu",
+                             validators=[validate_email])
+    password = models.CharField(max_length=30, validators=[MinLengthValidator(8)])
     date_created = models.DateTimeField()
     date_modified = models.DateTimeField()
-    
+
+
 class Game(models.Model):
     GAME_RESULT = (
```

Comment:
This assignment looks ambiguous to me. Could you clarify what you mean by 'Y' / 'Yes' being the result of a game? 
@otaiwo70 @rakhfa

Reply:
I'd also like to see this be a recording of what place the player got, not just "did they win"

Reply:
refers to section below specifying whether a user is a winner or not a weather or undecided if the game is in progress

is_winner = models.CharField(max_length=1, choices=GAME_RESULT)

Reply:
So far by design we are only tracking user's game statuses: win/lose/pending. If we want to track user's ranking in a game then we will have to change our db model accordingly. I will work on that.